app/services/recommendations/job.rb

```diff
@@ -0,0 +1,39 @@
+module Recommendations
+  class Job
+    attr_accessor :job, :user
+
+    def initialize(job)
+      self.job = job
+    end
+
+    def for(user)
+      self.user = user
+      self
+    end
+
+    def similar_jobs(limit = 2)
+      return [] unless job
+
+      limit = 2 if limit <= 0
+
+      job_query = job.query_as(:job).
+        # match Jobs which received applications from users who also also applied to the given job
+        match("(job:Job) <-[:APPLIED_TO]- (similarUser:User) -[:APPLIED_TO]-> (recommendedJob:Job)").
+        break.
```

Comment:
Just discovered that

```
match
    (job:Job) <-[:APPLIED_TO]- (similarUser:User) -[:APPLIED_TO]-> (recommendedJob:Job),
    (recommendedJob) <-[applications:APPLIED_TO]- ()
```
is not the same as
```
match
    (job:Job) <-[:APPLIED_TO]- (similarUser:User) -[:APPLIED_TO]-> (recommendedJob:Job)
match
    (recommendedJob) <-[applications:APPLIED_TO]- ()
```

We need the `break` to produce 2 `match` statements.